sdk/containerregistry/azure-containers-containerregistry/src/main/java/module-info.java

```diff
@@ -7,12 +7,9 @@
     exports com.azure.containers.containerregistry;
     exports com.azure.containers.containerregistry.models;
 
-    exports com.azure.containers.containerregistry.implementation to com.azure.core;
-    exports com.azure.containers.containerregistry.implementation.models to com.azure.core;
-
-    // exporting some packages specifically for Jackson
-    opens com.azure.containers.containerregistry to com.fasterxml.jackson.databind, com.azure.core;
     opens com.azure.containers.containerregistry.implementation to com.fasterxml.jackson.databind, com.azure.core;
     opens com.azure.containers.containerregistry.implementation.models to com.fasterxml.jackson.databind, com.azure.core;
-    opens com.azure.containers.containerregistry.models to com.fasterxml.jackson.databind, com.azure.core;
+
+    // This is still needed even though we export the models.
+    opens com.azure.containers.containerregistry.models to com.fasterxml.jackson.databind;
```

Comment:
Does this need to be opened to `com.azure.core`?

Reply:
I did not see any issues while running my tests. But thanks for reminding - I believe it was you who mentioned core might need this too? Do you remember in what scenarios it needed it? To be safe, I can expose but would like to know why.

Also, since this package is already exported, opens only ensures that some of the model private ctors can be called. Everything else here should be already callable.